packages/sources/finage/src/adapter.ts

```diff
@@ -53,12 +51,16 @@ export const execute: Execute = async (input) => {
     }
   }
 
-  const config = {
+  const options = {
```

Comment:
```suggestion
  const options = {
  ...config.api,
```

Could clean up the usage of `baseUrl` above.